### ISSUE TITLE
Snake Example is broken

### ISSUE THREAD

**Issue by jreamy** [toxoidengine/toxoid#4]
I cloned the repo, installed the various dependencies & submodules and tried to run the snake example, getting a long list of errors. I think it's probably related to this change https://github.com/toxoidengine/toxoid/commit/32177c9dd8c58afe194dec3e60990eaeed653ff5 but there may be other related changes that have left the snake example broken.

Or there's something wrong with my installation, but I thought I'd bring this up.

**Comment by jreamy** [comment]
I see the comment in the discord now
> The snake example is broken for now (maybe, haven't checked) because I've made many API changes since I last wrote it.

**Comment by troyedwardsjr** [comment]
@jreamy Yes, apologies. Since the WASM component model overhaul has been merged in though, instructions to run the snake example have been update: https://github.com/toxoidengine/toxoid/pull/6
